ramp.yml

```diff
@@ -22,3 +22,5 @@ capabilities:
     - intershard_tls
     - intershard_tls_pass
     - flash
+exclude_commands:
+    - ts._replicate
```

Comment:
Shouldn't it be `ts._replicate_auxLoad_before` and `ts._replicate_auxLoad_after`?